tests/python/gaia-ui-tests/gaiatest/tests/functional/contacts/test_import_edit_export_contact.py

```diff
@@ -68,6 +68,8 @@ def test_import_edit_export_contact(self):
         self.assertEqual(self.sim_contact['tel'][0]['value'], self.contact.tel['value'])
 
     def tearDown(self):
-        self.data_layer.delete_sim_contact(self.sim_contact['id'])
+        for contact in self.data_layer.sim_contacts:
+            if contact['name'][0].startswith('Name'):
+                self.data_layer.delete_sim_contact(contact['id'])
```

Comment:
I'd prefer to take no risk. A company might start with "name" and be in the SIM contacts.
We also should prepare for having more contributors.

We don't warn the user of his sim card being wiped. If we go on the wipe road, I'd make that clear on MDN and on the harness.